rebuild_fuzzer_corpus.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
```

Comment:
maybe you can add a docstring at the top of this script to explain what it is for, for when our future selves come across it